src/client/game.cpp

```diff
@@ -1238,10 +1238,11 @@ void Game::shutdown()
 	if (g_touchscreengui)
 		g_touchscreengui->hide();
 
-	showOverlayMessage(N_("Shutting down..."), 0, 0, false);
-
 	if (clouds)
 		clouds->drop();
+	
+	m_rendering_engine->run();
+	m_rendering_engine->draw_load_screen(wstrgettext("Shutting down..."), guienv, texture_src, -1, 0, true);
```

Comment:
I'm adding a comment here just to ensure it's visible: This change is required, otherwise the screen would turn black for the fraction of a second, which looks bad (in my opinion)!

Reply:
It's not actually necessary to do it like this, `showOverlayMessage(N_("Shutting down..."), 0, 0, true);` instead of `showOverlayMessage(N_("Shutting down..."), 0, 0, false);` gives the same result. However, I'm fine with the current change.